core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -40,11 +40,11 @@ export class AudioBarComponent {
   isPaused: boolean = true;
   directiveSubscriptions: Subscription = new Subscription();
   languagesInExploration: ExplorationLanguageInfo[];
-  audioBarIsExpanded: boolean;
-  progressBarIsShown: boolean;
+  audioBarIsExpanded!: boolean;
+  progressBarIsShown!: boolean;
   audioLoadingIndicatorIsShown: boolean = false;
   explorationPlayerModeIsActive: boolean;
-  selectedLanguage: { value: string };
+  selectedLanguage: { value: string | null };
```

Comment:
Why is `null` allowed here.

Reply:
Not sure. Ideally, we should get rid of nulls where possible. You will need to evaluate the code and see if the `null` is really needed.

Reply:
Another way of getting rid of the `null` here would be to throw an error whenever `selectedLanguage == null`. However, throwing an error here would break the code because this value is indeed sometimes `null`, and the code is supposed to work when it is `null` according to the tests. I think maybe `null` should be allowed here.

Reply:
Another way I can address this is to set `selectedLanguage.value` to `'en'` if `getCurrentAudioLanguageCode()` returns null. Unless there is a default string value to initialize `selectedLanguage.value` with, I'm not sure how to not allow null here without throwing an error. Would that work?

Reply:
In the code, I'm wondering whether "null" means something semantically. Is it actually possible in the UI for there to be "no selected language"?